opus/application/test_api/responses/results_nh_mvic_mc0_0032528036_files.json

```diff
@@ -15,6 +15,11 @@
                 "https://opus.pds-rings.seti.org/holdings/previews/NHxxMV_xxxx/NHJUMV_2001/data/20070131_003252/mc0_0032528036_0x536_sci_1_med.jpg",
                 "https://opus.pds-rings.seti.org/holdings/previews/NHxxMV_xxxx/NHJUMV_2001/data/20070131_003252/mc0_0032528036_0x536_sci_1_full.jpg"
             ],
+            "nh_mvic_documentation": [
+                "https://opus.pds-rings.seti.org/holdings/documents/NHxxMV_xxxx/Reuter-etal-2005-SPIE.pdf",
```

Comment:
There are several .png files in the NHMVIC documents directory that should be included here.